Source/UI/NewUI/PaymentForm/Model/PaymentStyle.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PaymentStyle.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 26/07/2022.
+//
+
+import Foundation
+
+public struct PaymentStyle {
```

Comment:
Is there a reason for this to be public, Do merchants use this for default UI set up? 
I can see that cardFormData is assigned in MainViewcontroller as  `Self.defaultCardFormData()` so it gets the style from the there. Why declare this public ?

Reply:
We previously had multiple objects passed into the factory to build the root vc for the journey. I took those objects and split them into 2 which should better guide consumer on setting up.

As such, these groups are:
- configuration (made of default billing data and other logical components)
- UI ☝️ (made of styling for the UI components)

I did not have time to refactor `Self.defaultCardFormData`, I can do that as well if this approach is accepted (no point to go back & forward on everything)

Reply:
This will also need to be added to API spec, @ehab-al-cko can you please add this to the API spec as well.